docs_app/content/tutorial/4-what-is-an-operator.md

```diff
@@ -0,0 +1,71 @@
+
+## Collections And Containers In JavaScript
+
+There are quite a few collections and containers already in existence in JavaScript. It's worth noting a few of them for comparisons here:
+
+### Arrays
+
+The most common collections of things we deal with in JavaScript are [Arrays](MDN_LINK_HERE). Arrays are collections that can be accessed synchronously, and randomly at indices. You can [map](MDN) an array into a new array of the same size. You can create an array of a smaller size (provided your current array isn't length of `0`), so it can be [filtered](MDN) to a new array. And arrays have a definitive "end" to them, which is represented in array literals with `]`, and at runtime by checking to see if your index is outside of the bounds provided by the `length`, therefor arrays can be [concatenated](MDN). Arrays can also be [flattened](MDN), as you can have an array of arrays.
+
+### Promises
+
+Yes, believe it or not, a promise is indeed a "collection". It's a collection of exactly one thing, no more no less. That means they're slightly more limited, but they can be mapped, with [`then`](MDN), concatenated (also `then`), and flattened (which is done automatically with, you guessed it... `then`). However, they can't be placed "inside" of one another. And they can't be "filtered", (there has to be one value or a rejection). So without even getting into their lack of cancellation, it's fair to say they're limited in terms of available operations.
+
+### Iterables
+
+There is a whole bit that we could go into here about how [Observable is the "dual" of Iterable](FIND_LINK), however, for brevity's sake: Iterables are [pull](GL)-based collections/containers of values, in which you can use an [Iterator](MDN) (or iteration via `for..of`) in order to [pull](GL) one value out at a time. Iterables can "contain" other Iterables. They can be any size. And Iterables signal their end by providing a `done` value in an [iteration result](MDN). They can be mapped, filtered, flattened, concatenated, et al. 
+
+If this topic interests, you, it's probably also worth looking at [async iterables](MDN), and libraries related to it, like [ixjs](https://github.com/reactivex/ixjs).
+
+### Observables
+
+Observables actually exist in the wild in various forms, but here we're obviously going to be talking about RxJS [`Observables`](API).
+
+Observables are a container ("opened via subscription") that holds values that are pushed over time. Observables [complete](GL), so you know when they've ended. Observables can also be of any size. Almost all operations that can be performed on the types above can also be performed on [`Observables`](API).
+
+## Operators
+
+You've made it this far, and you're probably thinking, "All of that is great... So what are operators?"
+
+An "operator" is sort of a blanket term for a thing that can perform an operation on a collection that results in a new collection. For example, `filter` and `map` are both "operators" for [`Array`](MDN).
+
```

Comment:
Consider adding something in between these two paragraphs that provide information on the purpose of operators with Observables before jumping into how they are implemented.

Something like "Operators on Observables transform each emitted item into a new Observable." Maybe that's not general enough. But it should be made clear that an operator on an Observable operates on each emitted item as it is emitted. (Unlike an array, a filter operator on an Observable filters as items are emitted, it doesn't wait for all items and then filter.)